src/OneDriveSdk/Requests/Extensions/IDriveSpecialCollectionRequestBuilderExtensions.cs

```diff
@@ -0,0 +1,15 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.OneDrive.Sdk
+{
+    public partial interface IDriveSpecialCollectionRequestBuilder
+    {
+        /// <summary>
+        /// Gets app root special folder item request builder.
+        /// <returns>The item request builder.</returns>
+        /// </summary>
+        IItemRequestBuilder AppRoot { get; }
```

Comment:
:/ just saw this was already in use under the other type name, we should enforce the break now while we are changing everything

---
In reply to: [73946373](https://github.com/OneDrive/onedrive-sdk-csharp/pull/141#discussion_r73946373) [](ancestors = 73946373)